en/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md

```diff
@@ -3,49 +3,158 @@ id: network-aruba-instant-snmp
 title: Aruba Instant SNMP
 ---
 
+## Overview
+
+Aruba Networks is a provider of network access solutions for enterprises.
+
+The Centreon Plugin-Pack *Aruba Instant SNMP* relies on the SNMP protocol to query and collect status and metrics of the Aruba Wireless Access Point equipments. 
+
+
+## Plugin-Pack assets
+
+### Monitored objects
+
+* Wireless Access Point
+
+### Collected metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Session-Usage-->
+
+| Metric name                     | Description                                  | Unit   |
+| :------------------------------ | :------------------------------------------- | :----- |
+| accesspoints.total.count        | Number of access points on the device        | Count  |
+| clients.current.count           | Number of clients connected to the device    | Count  |
+| cpu.utilization.percentage      | Used CPU on the device                       | %      |
+| memory.usage.percentage         | Percentage of memory usage of the device     | %      |
```

Comment:
Specify that this metric has an instance. 

Use: ap_name#cpu.utilization.percentage

Also add all metrics (memory.free.bytes and memory.usage.bytes are missing)